src/components/InputWithLabel.jsx

```diff
@@ -0,0 +1,25 @@
+import { useRef, useEffect } from 'react'
+
+
+export default function InputWithLabel(props) {
+  const { children, title, handleTitleChange } = props;
+  const inputRef = useRef(null)
+
+  useEffect(() => {
+    inputRef.current.focus()
```

Comment:
nit, you could run into a race condition error here. `inputRef` starts as `null`, then along the lifecycle, gets connected to the `input` element. so to account for that space of time when `inputRef` is null, I'd put a conditional before calling the `focus` method. something like 
```
if (inputRef?.current) inputRef.current.focus()

```


also, if you're getting no errors, great, but this is a useful thing to keep in mind

Reply:
Thanks, I always forget about the condition for refs.